### PR TITLE
Adaptions to run modules in gui

### PR DESCRIPTION
**Pull request by juleshaas**
MR
* makes changes to make modules usable in GUI
* makes smaller label changes
* updates black version
* adds pandas to docker image
* adds description of how to start docker on Windows
* lints README

### CODE REVIEW

**Review by anikaweinmann — APPROVED** [review]

File: grass-gis-addons/m.analyse.trees/v.trees.cd/v.trees.cd.py [hunk 1]
```diff
@@ -392,12 +408,13 @@ def main():
                 pid,
             )
         # remove unnecessary columns
-        grass.run_command(
-            "v.db.dropcolumn",
-            map=cd_output_i,
-            columns=rm_vec_columns,
-            quiet=True,
-        )
+        if grass.vector_db(cd_output_i):
```
Comment (linakrisztian) [inline grass-gis-addons/m.analyse.trees/v.trees.cd/v.trees.cd.py:411]:
```suggestion
        # only for non-empty vector (i.e. vector with attribute table)
        if grass.vector_db(cd_output_i):
```

File: grass-gis-addons/m.analyse.trees/v.trees.param/v.trees.param.py [hunk 2]
```diff
@@ -289,6 +289,7 @@ def main():
     if nprocs > 1:
         for subset_name, mapset_name in zip(subset_names, mapset_names):
             treecrown_subset_mapset.append(f"{subset_name}@{mapset_name}")
+        grass.message(len(mapset_names))
```
Comment (linakrisztian) [inline grass-gis-addons/m.analyse.trees/v.trees.param/v.trees.param.py:292]: Was it only for debugging? Otherwise extend message maybe a little bit.

**Review by linakrisztian — APPROVED** [review]
Just some small remarks

File: grass-gis-addons/m.analyse.trees/v.trees.param/v.trees.param.py (hunk 2 above)
Comment (anikaweinmann) [reply grass-gis-addons/m.analyse.trees/v.trees.param/v.trees.param.py]: only debugging

Comment (juleshaas) [reply grass-gis-addons/m.analyse.trees/v.trees.param/v.trees.param.py]: yes, I removed it.

**Pull request merged**